solr/core/src/test/org/apache/solr/internal/csv/CSVParserTest.java

```diff
@@ -377,37 +377,6 @@ public void testEmptyLineBehaviourCSV() throws Exception {
     }
   }
 
-  public void OLDtestBackslashEscaping() throws IOException {
```

Comment:
How about keeping this and renaming to `testOLDBackslashEscaping()` say?

Reply:
sure, I'll restore it, and move it below the first two `testBackslashEscaping`!

Reply:
Umm, it fails....    and since we were not running it, I'd rather not debug why it fails...  and just ignore it.  since maybe it fails cause we changed logic...